Sources/CoreAardvark/Logging/ARKLogStore.m

```diff
@@ -72,6 +71,28 @@ - (nullable instancetype)initWithPersistedLogFileName:(NSString *)fileName;
     return [self initWithPersistedLogFileName:fileName maximumLogMessageCount:2000];
 }
 
+- (nullable instancetype)initWithPersistedLogFileURL:(nonnull NSURL *)persistedLogFileURL maximumLogMessageCount:(NSUInteger)maximumLogMessageCount;
```

Comment:
should we funnel `initWithPersistedLogFileName:maximumLogMessageCount:` through to this? I think it'll be largely the same after the former builds a URL to use

Reply:
I did that at first, but I decided it would be better to leave `initWithPersistedLogFileName:maximumLogMessageCount:` as a designated initializer so subclasses can still call through without having to deal with finding the application support directory. Unfortunately the designated initializer needs to call into a `super` initializer, so it can't go through this one.